layers/+lang/extra-langs/packages.el

```diff
@@ -12,6 +15,9 @@
 (defun extra-langs/init-arduino-mode ()
   (use-package arduino-mode :defer t))
 
+(defun extra-langs/init-faust-mode ()
+  (use-package faust-mode :defer t :mode "\\.dsp\\'" :mode "\\.lib\\'"))
```

Comment:
Also, you can group the two `:mode` in one expression, like for instance [here](https://github.com/syl20bnr/spacemacs/blob/develop/layers/finance/packages.el#L28).